packages/api/app/Controllers/Http/TechnologyOrderController.js

```diff
@@ -0,0 +1,59 @@
+const { orderStatuses, errorPayload, errors } = require('../../Utils');
+
+const Config = use('Adonis/Src/Config');
+const Mail = require('../../Utils/mail');
+
+const TechnologyOrder = use('App/Models/TechnologyOrder');
```

Comment:
To fazendo os gets de orders na minha PR

Reply:
Blza @NaylsonFerreira aqui eu vou fazer apenas o fechamento do pedido.

Reply:
Já fiz isso, veja lá 👍